CODE_STANDARDS.txt

```diff
@@ -4,16 +4,16 @@ consistently such that we do not confuse each other. Also organizing the
 code will make it a lot easier to debug if we run into any issues or when
 we are correcting values during testing.
 
-- All tabs should be 4 spaces
+- Try to implement new features on branches that are not master, fixes are exempt.
+  This is important as code changes that are made straight to master cannot be properly reviewed.
+- All tabs should be 4 spaces. Indents should be made with space characters.
 - the curly brackets for functions should start on the same line as their
   parent function
 - variable names should mean something. Avoid using single letters even
   for loops and other iterative structures. Exceptions will be made if
   performance decreases significantly in certain areas of code.
-- all functions, if statements, loops etc should should have
-  an end comment: eg: //end if, //end main
 - add a empty line at the start of the function and the end
-- comments should be roughly 60-80 characters in line length
+- comments should be roughly 60-120 characters in line length
```

Comment:
If anyone wants to program on an 80 char terminal let the rest of us know.
I guess if @CrazymonkeyA17 is still using that tiny netbook this might be relevant, but even then I'm sure 120 chars *should* fit on the screen.